### DISCUSSION TITLE
Check for last backup age on startup

### DISCUSSION THREAD

**Discussion by jose1711** [borgbase/vorta#992] [Ideas]
Maybe your notification daemon is broken or you're in DND mode all the time, or perhaps just run the backups manually. For these situations it could be handy if `vorta` checks the age of your last backup (check is on by default) on startup and displays a warning if it reaches a configured threshold (e. g. 7 days w/o a backup). But maybe I'm just having paranoia combined with backup-OCD :smile:

**Comment by m3nu** [comment]
Kinda implemented in #908, but no time to finish it yet.
